ci-operator/config/openshift-kni/lifecycle-agent/openshift-kni-lifecycle-agent-main__nightly-4.15-upgrade-from-stable-4.14.yaml

```diff
@@ -0,0 +1,95 @@
+base_images:
+  lca-operator:
+    name: "4.15"
+    namespace: ocp-kni
+    tag: lifecycle-agent-operator
+  tests:
+    name: "4.16"
+    namespace: ocp
+    tag: tests
+releases:
+  latest:
+    integration:
+      name: "4.15"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 10m
+      memory: 100Mi
+tests:
+- as: image-based-upgrade-e2e-serial-conformance
+  cluster: build02
+  interval: 24h
+  steps:
+    cluster_profile: aws
+    dependencies:
+      LCA_PULL_REF: pipeline:lca-operator
+      TESTS_PULL_REF: pipeline:tests
+    env:
+      OCP_BASE_IMAGE_SOURCE: ci
+      OCP_BASE_VERSION: "4.15"
+      OCP_TARGET_IMAGE_SOURCE: release
+      OCP_TARGET_VERSION: "4.14"
+      SEED_IMAGE_TAG_FORMAT: e2e
+    workflow: openshift-image-based-upgrade-e2e
+  timeout: 4h0m0s
+- as: image-based-upgrade-e2e-serial-conformance-baseline
+  cluster: build02
+  interval: 24h
+  steps:
+    cluster_profile: aws
+    dependencies:
+      LCA_PULL_REF: pipeline:lca-operator
+      TESTS_PULL_REF: pipeline:tests
+    env:
+      CREATE_CLUSTER_ONLY: "true"
+      OCP_BASE_IMAGE_SOURCE: ci
+      OCP_BASE_VERSION: "4.15"
+      OCP_TARGET_IMAGE_SOURCE: release
```

Comment:
No target is created in the baseline jobs so you can skip setting all the target envs for these